shopinvader_customer_validate/models/res_partner.py

```diff
@@ -85,6 +91,19 @@ def _display_validate_address(self):
         has_parent = self._get_has_shopinvader_parent()
         return has_parent and self.address_type == "address"
 
+    def _compute_backends_display(self):
+        for rec in self:
+            rec.backends_display = ", ".join(rec._get_valid_backends().mapped("name"))
```

Comment:
Suggestion, to be agnostic about the field representing the record:
```suggestion
            rec.backends_display = ", ".join(rec._get_valid_backends().mapped("display_name"))
```